src/presentational-components/shared/pf4-select-wrapper.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import { FormSelect, FormSelectOption, FormGroup, TextContent, Text, TextVariants } from '@patternfly/react-core';
 
 const createOptions = (options, inputValue, isRequired) => {
-  if (inputValue && isRequired) {
+  if (inputValue) {
```

Comment:
How does this change affect existing selects? There should be one in order form and when adding/editing portfolio

Reply:
@Hyperkid123  - I had to make this change in order for the initial value to be shown.

Reply:
Well this is going to break all the screens with select. This is obviously a bug i will check what is going on but we can't just change it like this.

Reply:
What initial value its not showing?

Reply:
Removed the changes made to to this file.